### PR TITLE
RM-75189 RM-75188 Release over_react 3.6.0

### LINKED ISSUE #509: [DOCS] Finalize analyzer plugin readme
* This should be one of the very last things done before the initial public release.

Closely related to #508

### PR DESCRIPTION
**Pull request by greglittlefield-wf**
PRs merged to integration branch that still need a +1:
- [x] https://github.com/Workiva/over_react/pull/582
- [x] https://github.com/Workiva/over_react/pull/576 (just needs a +1 on my last two commits)

Documentation prep prep:
- ff69e6f Reduce size of doc images: make pngs indexed, use static image instead (this off 7MB)
- d25b51e Denote plugin as "beta", add fuller description, add more references in main REAME
- b55a40a Disable some assists/diagnostics for initial release

Fix issues found during final round of manual testing in different repos:
- 6919a37 Fix false positives in arrowFunctionProp when there's a descendant cascade
- 4fa1c2a Don't add an error when functions are passed as children
- 2682652 Fix/improve wording of messages

---

FIxes #509

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # OverReact Changelog
 
+## [3.6.0](https://github.com/Workiva/over_react/compare/3.5.3...3.6.0)
+
+- Add [OverReact Analyzer Plugin (beta)](https://github.com/Workiva/over_react/tree/link-2020-integration/tools/analyzer_plugin) ⚡️
```
Comment (aaronlademann-wf) [inline CHANGELOG.md:5]:
```suggestion
- Add [OverReact Analyzer Plugin (beta)](https://github.com/Workiva/over_react/tree/master/tools/analyzer_plugin) ⚡️
```

Comment (greglittlefield-wf) [reply CHANGELOG.md:5]: :doh:

File: .travis.yml [hunk 2]
```diff
@@ -16,13 +16,13 @@ jobs:
     # Analyzer Plugin CI Jobs
     - dart: "2.7.2"
       name: "Analyzer Plugin (SDK: 2.7)"
-      script: cd ./tools/analyzer_plugin && ./tool/travis.sh
+      script: dart travis_link_plugin_deps.dart && cd ./tools/analyzer_plugin && ./tool/travis.sh
```
Comment (greglittlefield-wf) [inline .travis.yml]: Whoops, forgot `tool/`. Addressed in d36673aa

**Comment by aaronlademann-wf** [comment]
QA +1

- [x] CI Passed
- [x] Version strings all look correct
- [x] Pulled down locally, spun up a project using the plugin - the plugin started as expected.

**Comment by aaronlademann-wf** [comment]
QA -1 

Found an issue with the `propTypes` static enforcement...

**Comment by greglittlefield-wf** [comment]
@Workiva/release-management-p

**Pull request merged**